app/server/logging/jsondb/src/main/java/io/syndesis/server/logging/jsondb/controller/KubernetesSupport.java

```diff
@@ -42,10 +44,12 @@
 
     private final KubernetesClient client;
     private final OkHttpClient okHttpClient;
+    /** Read timeout for the HTTP client, we expect to receive at least one log line in 30 mins. */
+    private Duration readTimeout = Duration.ofMinutes(35);
```

Comment:
Is this 30 mins for getting a logline configured somewhere ?

Reply:
No and it is just an assumption that every integration should perform some activity within 30min, so after 30min + 5min of grace period (all ball parked) if the client hasn't received any log lines we consider the connection to be half-open and we need to timeout and retry.

Reply:
I can imagine cron-job like integration which fire up only once a day or so for (batch processing), so while this is a good rule of thumb for now, I don't think that this assumption holds in general.